### PR TITLE
[eclipse/xtext#2042] package old generator separately

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext#2042] package old generator separately

### CODE REVIEW

**Comment by cdietrich** [comment]
this this the variant @kthoms prefers

File: releng/org.eclipse.xtext.generator.feature/feature.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+featureName=Xtext Old Deprecated Language Generator
```
Comment (szarnekow) [inline releng/org.eclipse.xtext.generator.feature/feature.properties:1]: > Xtext Old Deprecated Language Generator

Old is relative. Maybe just `Xtext Deprecated Language Generator`

File: releng/org.eclipse.xtext.generator.feature/feature.xml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+   </copyright>
+   <license url="http://www.eclipse.org/legal/epl-2.0">
+      %license
+   </license>
+   <includes
+         id="org.eclipse.xtext.sdk"
+         version="0.0.0"/>
+
+   <requires>
+      <import plugin="org.apache.log4j" version="1.2.15" match="equivalent"/>
```
Comment (szarnekow) [inline releng/org.eclipse.xtext.generator.feature/feature.xml:31]: That one still strictly needs log4j 1.2.15?

**Review by szarnekow — APPROVED** [review]

File: releng/org.eclipse.xtext.generator.feature/feature.xml (hunk 2 above)
Comment (cdietrich) [reply releng/org.eclipse.xtext.generator.feature/feature.xml:31]: no thus the equivalent 
> equivalent - dependent plug-in version must be at least at the version specified, or at a higher service level (major and minor version levels must equal the specified version).

File: releng/org.eclipse.xtext.generator.feature/feature.properties (hunk 1 above)
Comment (cdietrich) [reply releng/org.eclipse.xtext.generator.feature/feature.properties]: fixed

File: releng/org.eclipse.xtext.generator.feature/feature.xml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      provider-name="%providerName"
+      license-feature="org.eclipse.license"
+      license-feature-version="0.0.0">
+
+   <description>
+      %description
+   </description>
+
+   <copyright>
+      Copyright (c) 2009, 2022 itemis AG (http://www.itemis.eu) and others.
```
Comment (kthoms) [inline releng/org.eclipse.xtext.generator.feature/feature.xml:15]: This is initially added now, so just `Copyright (c) 2022 itemis AG`

Comment (cdietrich) [reply releng/org.eclipse.xtext.generator.feature/feature.xml]: done

**Review by kthoms — APPROVED** [review]

**Pull request merged**